Wishlist dropin (#347)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/wishlist/containers/wishlist-alert.mdx

---
title: WishlistAlert container
description: Learn about the WishlistAlert container.
---

import Diagram from '@components/Diagram.astro';
import OptionsTable from '@components/OptionsTable.astro';

> The WishlistAlert container displays notifications giving feedback about the different actions that can be done in a wishlist.

Reviewer: ```suggestion
A WishlistAlert container displays notifications that give feedback about actions performed in a Wishlist drop-in.
```
